webrtc.html

```diff
@@ -5957,6 +5957,10 @@ <h3 id="sec-constraints">Constraints</h3>
     Haza&euml;l-Massieux, for their support. Substantial text in this
     specification was provided by many people including Martin Thomson, Harald
     Alvestrand, Justin Uberti, and Eric Rescorla.</p>
+
+    <p>The RTCRtpSender and RTCRtpReceiver objects were initially conceived in the
```

Comment:
Can we use the word "described" here?
The conception of these objects seems to have a lot more murky history, but the description is clearly based on the one in ORTC.